feeds/wlan-ap/opensync/src/src/sm/src/ubus_collector.c

```diff
@@ -126,13 +126,13 @@ static int frequency_to_channel(int freq)
 static radio_type_t frequency_to_band(int freq)
 {
 	int chan = frequency_to_channel(freq);
-
+	/*Need to add support for differentiating between 5G,5GU and 5GL*/
```

Comment:
This function is also used for the Client events: authenticate, associate, and disconnect.
Did you intend to change their behaviour?
Did they have the same problem?
Did you test that this better for them?

Reply:
I assumed that since we are supporting 2 radio device only for release 1 it would be same for client events, although some regression testing will be required. If my assumption is incorrect we will have to make 2 separate functions although in case of client events we would face the same problem on 2 radio devices that radios will be reported as 5GU/5GL no 5G.